pyproject.toml

```diff
@@ -64,6 +64,7 @@ flake8-isort = "^4.0.0"
 flake8-black = "^0.2.1"
 django = {version = ">=2,<4", optional = false}
 pydantic = {version = "^1.6.1", optional = false}
+email-validator = {version = "^1.1.2", optional = false}
```

Comment:
If someone is using pydantic with pydantic.networks.EmailStr they will need to install `email-validator` anyway otherwise pydantic will fail

So I think we should be fine including it only in the dev-dependencies for when we run the tests?

Reply:
@patrick91 👋

Reply:
I think it should be a non-dev dependency, or optional one, depending how we currently treat pydantic. Marking as a dev only sounds risky to me.

Reply:
this is fine :)

Reply:
@ulgens it is a dependency only used when using pydantic and the email field, in their doc they say you need to install email validator: https://pydantic-docs.helpmanual.io/usage/types/#pydantic-types

so for us it is ok to only have it in dev deps (because we use the field in tests), I don't think it is need to mark it as optional as Pydantic doesn't either

Reply:
Okay then.